CleanWeather/Scenes/FavouriteCities/FavouriteCitiesPresenter.swift

```diff
@@ -9,17 +9,35 @@
 import UIKit
 
 protocol FavouriteCitiesPresenter: SpinnerPresenter, AlertPresenter {
-    func presentCities(allCities: [City], favourites: [City])
+    func presentCities(allCities: [City], favourites: [City], cityName: String, filteringState: Bool)
+}
+
+extension FavouriteCitiesPresenter {
+    func presentCities(allCities: [City], favourites: [City], cityName: String = "", filteringState: Bool = false) {
+        presentCities(allCities: allCities, favourites: favourites, cityName: cityName, filteringState: filteringState)
+    }
 }
 
 final class FavouriteCitiesPresenterImpl<T: FavouriteCitiesPresentable>: SharedPresenter<T> {}
 
 extension FavouriteCitiesPresenterImpl: FavouriteCitiesPresenter {
 
-    func presentCities(allCities: [City], favourites: [City]) {
+    func presentCities(allCities: [City], favourites: [City], cityName: String, filteringState: Bool) {
```

Comment:
why not using 1 argument:
filteringPhrase: String?
which also determines filering state? if nil = you know that it's false
use 1 parameter, not 2 everywhere